### PR TITLE
[WF-638] Props default value

### PR DESCRIPTION
**Pull request by ixTec**
# Adding default value for props to the documentation prop table

### 🔧 Minor Changes

- Props that have default values now have a second comment with a `@default <value>` structure (e.g. `@default medium`)
- Default value is now shown in props table under the prop description

### ❓ Misc Changes

- Updated wording of various parts of docs to correct grammar / punctuation
- References to "the default 'medium' size" within prop descriptions have been replaced with "the default size", as the default value is now evident from context

## Screenshot(s):
Screenshot 2022-08-23 at 10 14 06

### CODE REVIEW

File: src/form-field/form-field.tsx [hunk 1]
```diff
@@ -11,17 +11,23 @@ import Description from './description';
 type FormFieldProps = {
   /* The label displayed above the input. */
   label: string;
-  /* Input component to be enhanced. In general pass Waffles [Input](/components/input), [TextArea](/components/text-area) or [Select](/components/select). Must be single element. */
+  /* Input component to be enhanced. In general, pass Waffles [Input](/components/input), [TextArea](/components/text-area) or [Select](/components/select). Must be a single element. */
   children: JSX.Element;
-  /* Defines the size of the form field. In most cases default `medium` size should be used. Children of the form field will inherit the size by default. */
+  /* Optional input identifier. Pass it when component is rendered on the server to avoid client-server attribute mismatch. */
+  id?: string;
```
Comment (blurbyte) [inline src/form-field/form-field.tsx:17]: I think it's some kind leftover. `id` is no longer needed after `useId` got updated.

File: doc-site/components/props-table.tsx [hunk 2]
```diff
@@ -93,6 +111,16 @@ function PropsTable({ metadata, isPolymorphic = false }: PropsTableProps) {
                   ) : (
                     '—'
                   )}
+                  {singleProp.defaultValue && (
+                    <div css={defaultValueWrapperStyle}>
+                      <Badge css={defaultValueBadgeStyle} variant="grey">
```
Comment (blurbyte) [inline doc-site/components/props-table.tsx:116]: Actually after seeing it live I think `grey` might be too much 'in your face', and draws too much attention. Probably `greySubtle` will be better here.

**Review by blurbyte — CHANGES_REQUESTED** [review]
Looks great, good job 👍. Just 2 minor things to address.

File: doc-site/pages/components/button.mdx [hunk 3]
```diff
@@ -25,7 +25,7 @@ import Inverted from '../../examples/button/inverted';
 
 The button comes in four different variants: `primary`, `secondary`, `plain`, `destructive`, and `upgrade`.
 
-**Primary** button ideally is used only once on a page to display the most important action we want the user to take. Usually, the `medium` size is sufficient, but to draw more attention `large` size can be used. Use `small` size sparingly, in the tight spaces of more complex user interfaces.
+**Primary** button ideally is used only once on a page to display the most important action we want the user to take. Usually, the default size is sufficient, but to draw more attention `large` size can be used. Use `small` size sparingly - in the tight spaces or more complex user interfaces.
```
Comment (blurbyte) [inline doc-site/pages/components/button.mdx:28]: I think somewhere in content guidelines there was paragraph abut _dash_ usage. As far as I remember they suggested to use _&mdash;_ and don't put spaces around it: ..size sparingly&mdash;in the tight..

**Review by blurbyte — CHANGES_REQUESTED** [review]

File: doc-site/pages/components/button.mdx (hunk 3 above)
Comment (ixTec) [reply doc-site/pages/components/button.mdx:28]: Ah good to know, cheers!

**Review by blurbyte — APPROVED** [review]

**Pull request merged**